doc/source/overview/faq.md

```diff
@@ -0,0 +1,56 @@
+# Frequently Asked Questions
+
+## General troubleshooting
+
+**Q. I'm getting code errors using a method on my model and my data, what to do?**
+
+**A.** There can be many reasons why the method does not work. For code exceptions it is a good idea to check the following:
+ - Read the [docstrings](../api/modules.rst) of the method, paying close attention to the [type hints](https://docs.python.org/3/library/typing.html) as most errors are due to misformatted input arguments.
+ - Check that your model signature (it's type and expected inputs/outputs) are in the right format. For further details refer to [White-box and black-box models](../overview/white_box_black_box.md).
+
+**Q. The explanations are taking a long time to complete, what to do?**
+
+**A.** Explanations can take different times as a function of the model, the data and the explanation type itself. Refer to the [Introduction](../overview/high_level.md) and Methods sections for notes on algorithm complexity. You might need to experiment with the type of model, data points (specifically feature cardinality) and method parameters to ascertain if a specific method scales well for your use case.
+
+**Q. The explanation returned doesn't make sense to me, what to do?**
+
+**A.** Explanations reflect the decision-making process of the model and not that of a biased human observer, see [Biases](../overview/high_level.md#biases). Moreover, depending on the method, the data and the model, the explanations returned are valid but may be harder to interpret (e.g. see [Anchor explanations](#anchor-explanations) for some examples).
+
+**Q. Is there a way I can get more information from the library during the explanation generation process?**
+
+**A.** Yes! We use [Python logging](https://docs.python.org/3/howto/logging.html) to log debug messages from the library. You can configure logging for your script to see these messages during the execution of the code. Additionally, some methods also expose a `verbose` argument to print information to standard output.
```

Comment:
Maybe we could be more explicit about the recommended way to change the logging levels? i.e. do we recommend most users just passing `--log=DEBUG` as an arg to their script? Or does calling `logger = logging.get_logger(<explainer class>)` and then `logger.setLevel(logging.DEBUG)` work, in order to enable logging levels to be changed at run time (in a notebook for e.g.).

Reply:
I believe the cli arguments need parsing and feeding into the logging config by the application writer anyway. In general, setting up Python logging is pretty non-intuitive, but perhaps we could suggest the usage of `logging.basicConfig` to set the level globally?

Reply:
I've added an example of configuring using `logging.basicConfig` with the caveats. On second look, we don't actually use that much logging across the library, so the section is kind of moot.

Reply:
True... We should probably aim to start to add more logging going forward (and replace `print`'s with logging where applicable?), so this is nice to have I think. Do we comment out for now though?

Reply:
A nice principle would be that whenever we need to use a debugger to debug some user issue, think if we can include some intelligent lines of logging that could have caught the issue without going through a debugger.